.github/workflows/release.yml

```diff
@@ -0,0 +1,68 @@
+name: Release
+on:
+  push:
+    tags:
+      - v*
+
+env:
+  ELIXIR_OPTS: "--warnings-as-errors"
+  ERLC_OPTS: "warnings_as_errors"
+  LANG: C.UTF-8
+
+jobs:
+  create_draft_release:
+    runs-on: ubuntu-18.04
+    env:
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 50
+      - name: Create draft release
+        run: |
+          gh release create --title ${{ github.ref_name }} --notes '' --draft ${{ github.ref_name }}
```

Comment:
I think we may use repo name from `${{ github.repository }}` to push directly to the target repository instead of cloning a repo.